### PR TITLE
Cmr 7364

### PR DESCRIPTION
**Pull request by littleidiot40**
A reference to a related issue ticket number

A description of the changes proposed in the pull request.

How to test said change

@mentions of the person or team responsible for reviewing proposed changes or are stake holders in the change

### CODE REVIEW

**Comment by littleidiot40** [comment]
runme.sh -p -i
Will fail unless

The dist directory exists at CMR/python - added to repo
pip3 install wheel is executed first - documented

File: CMR/python/demos/notebooks/collections.ipynb [hunk 1]
```diff
@@ -27,9 +29,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```
Comment (jceaser) [inline CMR/python/demos/notebooks/collections.ipynb:34]: outputs need to be cleared before committing

File: CMR/python/demos/notebooks/collections.ipynb [hunk 2]
```diff
@@ -234,9 +248,8 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
-   "name": "python3"
+   "name": "python2716jvsc74a57bd0545e036c4b32438aced1f6b3c8d38ca151d9c36189e05839cb0aa568fda70ddd",
+   "display_name": "Python 2.7.16 64-bit"
```
Comment (jceaser) [inline CMR/python/demos/notebooks/collections.ipynb:252]: it just hit me, your on python 2?

**Review by jceaser — CHANGES_REQUESTED** [review]

File: CMR/python/demos/notebooks/collections.ipynb (hunk 1 above)
Comment (littleidiot40) [reply CMR/python/demos/notebooks/collections.ipynb:34]: Not sure how that got into the PR. Retracting.

**Comment by jceaser** [comment]
can you remove .dist as well?

**Comment by littleidiot40** [comment]
> can you remove .dist as well?

This is how you get an empty directory into a git repo.

https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

**Review by jceaser — APPROVED** [review]
having the dist directory is explained, will make compile smother for end user if they don't have to create it. runme.sh is more for documentation purposes, repository should be as easy to use as possible.

**Review by charleshuang80 — APPROVED** [review]

**Pull request merged**